fanuc_lrmate200ic5h_moveit_config/launch/move_group.launch

```diff
@@ -5,15 +5,16 @@
   <!-- GDB Debug Option -->
   <arg name="debug" default="false" />
   <arg unless="$(arg debug)" name="launch_prefix" value="" />
-  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />
+  <arg     if="$(arg debug)" name="launch_prefix"
+	   value="gdb -x $(find fanuc_lrmate200ic5h_moveit_config)/launch/gdb_settings.gdb --ex run --args" />
```

Comment:
O wait. Apparently it *does* exist (or at least, is supposed to exist): see [ros-planning/moveit_setup_assistant/commit/ae031777](https://github.com/ros-planning/moveit_setup_assistant/commit/ae031777f041d9c57a9c09db88f102cbe7ee35d3).

Not sure why it didn't end up in the commits though.

I'll check.